dist/file-preview-card/file-preview-card.css

```diff
@@ -10,7 +15,7 @@
     );
     box-sizing: border-box;
     display: grid;
-    grid-template-areas: " . . menu-action " " . center-action . " " label . . ";
+    grid-template-areas: " . . action " " . counter . " " signal . . ";
```

Comment:
Are we going to stick with grid or go to using position absolute?

Reply:
I just switched to absolute positioning, I'll change some names and then we should be ready for review